src/ansys/fluent/core/services/app_utilities.py

```diff
@@ -0,0 +1,402 @@
+    ) -> AppUtilitiesProtoModule.IsSolutionDataAvailableResponse:
+        """Is solution data available RPC of AppUtilities service."""
+        return self._stub.IsSolutionDataAvailable(request, metadata=self._metadata)
+
+    def register_pause_on_solution_events(
+        self, request: AppUtilitiesProtoModule.RegisterPauseOnSolutionEventsRequest
+    ) -> AppUtilitiesProtoModule.RegisterPauseOnSolutionEventsResponse:
+        """Register on pause solution events RPC of AppUtilities service."""
+        return self._stub.RegisterPauseOnSolutionEvents(
+            request, metadata=self._metadata
+        )
+
+    def resume_on_solution_event(
+        self, request: AppUtilitiesProtoModule.ResumeOnSolutionEventRequest
+    ) -> AppUtilitiesProtoModule.ResumeOnSolutionEventResponse:
+        """Resume on solution event RPC of AppUtilities service."""
+        return self._stub.ResumeOnSolutionEvent(request, metadata=self._metadata)
+
+    def unregister_pause_on_solution_events(
+        self, request: AppUtilitiesProtoModule.UnregisterPauseOnSolutionEventsRequest
+    ) -> AppUtilitiesProtoModule.UnregisterPauseOnSolutionEventsResponse:
+        """Unregister on pause solution events RPC of AppUtilities service."""
+        return self._stub.UnregisterPauseOnSolutionEvents(
+            request, metadata=self._metadata
+        )
+
+    def exit(
+        self, request: AppUtilitiesProtoModule.ExitRequest
+    ) -> AppUtilitiesProtoModule.ExitResponse:
+        """Exit RPC of AppUtilities service."""
+        return self._stub.Exit(request, metadata=self._metadata)
+
+
+class AppUtilitiesOld:
+    """AppUtilitiesOld."""
+
+    def __init__(self, scheme_eval):
+        """__init__ method of AppUtilitiesOld class."""
+        self.scheme_eval = scheme_eval
+
+    def get_product_version(self) -> str:
+        """Get product version."""
+        return self.scheme_eval.version
+
+    def get_build_info(self) -> dict:
+        """Get build info."""
+        build_time = self.scheme_eval.scheme_eval("(inquire-build-time)")
+        build_id = self.scheme_eval.scheme_eval("(inquire-build-id)")
+        vcs_revision = self.scheme_eval.scheme_eval("(inquire-src-vcs-id)")
+        vcs_branch = self.scheme_eval.scheme_eval("(inquire-src-vcs-branch)")
+        return {
+            "build_time": build_time,
+            "build_id": build_id,
+            "vcs_revision": vcs_revision,
+            "vcs_branch": vcs_branch,
+        }
+
+    def get_controller_process_info(self) -> dict:
+        """Get controller process info."""
+        cortex_host = self.scheme_eval.scheme_eval("(cx-cortex-host)")
+        cortex_pid = self.scheme_eval.scheme_eval("(cx-cortex-id)")
+        cortex_pwd = self.scheme_eval.scheme_eval("(cortex-pwd)")
+        return {
+            "hostname": cortex_host,
+            "process_id": cortex_pid,
+            "working_directory": cortex_pwd,
+        }
```

Comment:
@seanpearsonuk @mkundu1 

AppUtilities and AppUtilitiesOld are consistent with interface,  signature of methods. AppUtilities uses AppUtilitiesService and AppUtilitiesOld uses scheme_eval service, both are consistent in the output.

Reply:
Awesome, thanks, @hpohekar. Is there a test in place to show that we are consistent before and since 25R2?

Reply:
@seanpearsonuk Actually the existing tests are already verifying this as we are using Fluent version markers and codegen step for each version.

Reply:
@seanpearsonuk We have added tests for app utilities.